packages/webpack/README.md

```diff
@@ -8,28 +8,127 @@ We want to provide a good set of defaults on how to facilitate your web componen
+To use the default configuration:
+
+1. Install the required dependencies:
+```bash
+npm i -D @open-wc/webpack webpack webpack-dev-server http-server
+```
+
+2. Create a file `webpack.config.js`:
+```javascript
+const path = require('path');
+const defaultConfig = require('@open-wc/webpack/default-config');
+
+module.exports = defaultConfig();
 ```
+
+3. Add an `index.html` with at least the webcomponents-loader for loading the polyfills:
+```html
+<!doctype html>
+<html>
+  <head></head>
+  <body>
+    <script src="./node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+  </body>
+</html>
+```
+
+4. Add a `index.js` which kickstarts your JS code
+
+5. Add the following commands to your `package.json`:
+```json
+{
+  "scripts": {
+    "start": "webpack-dev-server --mode development --open",
+    "start:es5": "webpack-dev-server --mode development --open --es5",
+    "start:build": "http-server dist/ -o",
+    "build": "rm -rf dist && webpack --mode production"
+  }
+}
+```
+- `start` will run your app for development, it works only browsers which supports `es2015` for faster builds
+- `start:es5` will run your for development, it works also on browsers which support `es5`
+- `build` will build your app for production and output it in the `/dist` folder
+- `start:build` will run your app using a plain web server, to prove it works without magic :)
+
+
+### Notes
+To change the default `index.html` and `index.js` files:
+
+```javascript
+module.exports = defaultConfig({
+  indexHTML: path.resolve(__dirname, './src/index.html'),
+  indexJS: path.resolve(__dirname, './src/index.js'),
+});
+```
+
+## Features
+
+### Module resolution
+This lets you use bare imports: `import { bar } from 'foo'` instead of `import { bar } from '../node_modules/foo/foo.js'`
+
+### Automatic module type selection
+Based on `package.json` it selects the most optimal type of module format. es modules and es2015 code is preferred, but falls back to support other module types such as CommonJS and UMD. These can be a useful way to get done writing code today, but these are not future proof formats so be aware when importing non-es module dependencies.
+
+### HTML, JS and CSS minifications
+JS is minified for performance. HTML, CSS and is minified when they are used in combination with the `html` and `css` template tags from `lit-html` and `lit-element`.
+
+### es2015 and es5 output
+Using [webpack-babel-multi-target-plugin](https://www.npmjs.com/package/webpack-babel-multi-target-plugin), our build outputs an es5 and es2015 version of your app. Using the `nomodule` trick, we can serve es2015 code on modern browsers and es5 on older browsers (IE11 mostly). This significantly recudes the size of your app on modern browsers.
+
+Read more about this trick here: https://jakearchibald.com/2017/es-modules-in-browsers/
+
+For modern browsers code is compiled to es2015, for older browsers (IE11 mostly) there is an es5 output. Using the `nomodule`
```

Comment:
```suggestion
```

Reply:
it's the same info as 2 sentences before?

Reply:
Ah, yes :)